exec/totemconfig.c

```diff
@@ -450,6 +451,14 @@ static int totem_get_crypto(struct totem_config *totem_config, const char **erro
 		tmp_model = "nss";
 	}
 
+	if (icmap_get_string("totem.secauth", &str) == CS_OK) {
+		if (strcmp(str, "on") == 0) {
+			totem_config->secauth = 1;
+			tmp_cipher = "aes256";
+			tmp_hash = "sha256";
+		}
```

Comment:
`icmap_get_string` allocates new string which must be freed, so please add `free(str);`

Reply:
will include, thanks!

Reply:
included in v2